Week3/homework/step3-bonus.js

```diff
@@ -1,14 +1,26 @@
 'use strict';
 
 const values = ['a', 'b', 'c', 'd', 'a', 'e', 'f', 'c'];
+const myColors = ['red', 'blue', 'green', 'blue', 'brown', 'yellow', 'red'];
+const myNumbers = [5, 8, 12, 56, 8, 40, 5];
 
 function makeUnique(arr) {
-  // Replace this comment and the next line with your code
-  console.log(arr);
+  // eslint-disable-next-line no-shadow
+  return arr.filter((value, index, arr) => arr.indexOf(value) === index);
 }
 
-const uniqueValues = makeUnique(values);
-console.log(uniqueValues);
+console.log(makeUnique(values));
+console.log(makeUnique(myColors));
+console.log(makeUnique(myNumbers));
+
+function unique(arr) {
+  return [...new Set(arr)];
+}
+console.log(unique(values));
+console.log(unique(myColors));
+console.log(unique(myNumbers));
+
+// I ve learned the both methods from stack overflow
```

Comment:
Both solutions are fine. It's okay to use stack overflow but make sure you can explain how the code works.